README.md

```diff
@@ -1,183 +1,18 @@
-# mainframer
+## mainframer 2.x
 
-Blip-blop, it's `mainframer`, simple (at the moment lol) script that allows you to move build process from your local machine to some remote one. This way you can free up your local machine for better things — like editing source code in your IDE without lags, freezes and running into swap, being able to actually _use_ your computer when the build is happening somewhere else.
+> TL;TR: sync project to remote machine, execute commands, sync project back.
 
-**For now** `mainframer` implemented to work with [Gradle][Gradle] (mostly Android-focused) projects, but its design is pretty universal, so we’re open for other things like [`go`][Go] in particular, but please file an issue before contributing support for other build systems, thanks! 
+**Table of content:**
```

Comment:
All items will be anchors to README's sections, I vote for not having wiki pages since files like README are in sync with project git state and easy to contribute to via pull requests.